From f845cd982ebd28d9107a252c0e3cf655927021a7 Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Sat, 26 Jan 2019 00:39:26 +0100
Subject: [PATCH] modules: update OpenWrt

6e16dd1234 mt76: update to the latest version
76037756d0 kernel: bump 4.14 to 4.14.94
455bfd1065 kernel: bump 4.9 to 4.9.151
fafd7691e6 opkg: update to latest Git head
e789bd2243 opkg: drop argument from check_signature in opkg.conf
3603c2321d ramips: mt7621: fix 5GHz WiFi LED on ZBT WG3526
7f98cd8d50 odhcpd: fix onlink IA check (FS#2060)
abd0f7995e kmod-sched-cake: bump to latest cake

Compile-tested: ar71xx-{tiny,generic}, ramips-mt7621, x86-64
---
 modules                                       |  2 +-
 ...7621-fix-5GHz-WiFi-LED-on-ZBT-WG3526.patch | 23 -------------------
 2 files changed, 1 insertion(+), 24 deletions(-)
 delete mode 100644 patches/openwrt/0012-ramips-mt7621-fix-5GHz-WiFi-LED-on-ZBT-WG3526.patch

diff --git a/modules b/modules
index 2ff89a53..f1217826 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing luci gluon'
 
 OPENWRT_REPO=https://git.openwrt.org/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-18.06
-OPENWRT_COMMIT=939fa07b041fef58196fba8dd4b5184adb7b4d3f
+OPENWRT_COMMIT=6e16dd1234ea4814af2b46a22b4aea111baef3ab
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-18.06
diff --git a/patches/openwrt/0012-ramips-mt7621-fix-5GHz-WiFi-LED-on-ZBT-WG3526.patch b/patches/openwrt/0012-ramips-mt7621-fix-5GHz-WiFi-LED-on-ZBT-WG3526.patch
deleted file mode 100644
index aa23268e..00000000
--- a/patches/openwrt/0012-ramips-mt7621-fix-5GHz-WiFi-LED-on-ZBT-WG3526.patch
+++ /dev/null
@@ -1,23 +0,0 @@
-From: Andreas Ziegler <dev@andreas-ziegler.de>
-Date: Thu, 27 Dec 2018 15:02:41 +0100
-Subject: ramips: mt7621: fix 5GHz WiFi LED on ZBT WG3526
-
-This fixes the 5GHz WiFi LED which was previously not working.
-
-Signed-off-by: Andreas Ziegler <dev@andreas-ziegler.de>
-
-diff --git a/target/linux/ramips/dts/ZBT-WG3526.dtsi b/target/linux/ramips/dts/ZBT-WG3526.dtsi
-index 104a51f8de7110c0f65f760af70033853f3da745..e42ec5e33de7fb69a9c668406f876f6443d2912b 100644
---- a/target/linux/ramips/dts/ZBT-WG3526.dtsi
-+++ b/target/linux/ramips/dts/ZBT-WG3526.dtsi
-@@ -101,6 +101,10 @@
- 			mediatek,mtd-eeprom = <&factory 0x8000>;
- 			ieee80211-freq-limit = <5000000 6000000>;
- 		};
-+
-+		led {
-+			led-sources = <2>;
-+		};
- 	};
- };
- 
-- 
2.17.1

